sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -39,6 +39,9 @@
       --add-opens com.azure.security.attestation/com.azure.security.attestation.models=ALL-UNNAMED
       --add-opens com.azure.security.attestation/com.azure.security.attestation.models=com.fasterxml.jackson.databind
     </javaModulesSurefireArgLine>
+    <codesnippet.skip>false</codesnippet.skip>
+    <javadocDoclet></javadocDoclet>
+    <javadocDocletOptions></javadocDocletOptions>
```

Comment:
Having empty tags seems a bit odd and is done in almost all libraries. Can we invert this pattern where libraries having a non-empty value would override what's in parent pom? (Of course, this might be easier after majority of the libraries have migrated to the new tool)

Reply:
These empty lines will go away once the codesnippet tooling transition is complete. They're empty as the configuration in the parent POM is to enable the previous codesnippet doclet and this empty configuration disables it.